common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -445,9 +444,31 @@ def test_default_cohort_assignment_settings(self):
 
         self.assertTrue(self.cohort_management_page.is_assignment_settings_disabled)
 
-        message = "There must be one cohort to which students can be randomly assigned."
+        message = "There must be one cohort to which students can automatically be assigned."
         self.assertEqual(message, self.cohort_management_page.assignment_settings_message)
 
+    def test_cohort_enable_disable(self):
```

Comment:
This test is really only testing disabling of cohorts. I'd add a similar test that verifies that cohorts can be enabled when previously disabled. You could also extend this test to include those cases, but my preference is to keep tests small and focused.